src/ex1/index_ex1.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Task Manager</title>
+    <link rel="stylesheet" href="style_ex1.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Dosis:wght@300;400;500;600&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="white-box">
+      <h1 class="main-header">Task Manager</h1>
+      <div class="task-input-wrapper">
+        <input class="task-input" type="text" placeholder="Add your new task" />
+        <button id="add-task-button" class="task-button hithere">
+          Add task
+        </button>
+      </div>
+      <ul class="task-list">
+        <div id="triangle" class="triangle"></div>
+        <div id="box" class="box"></div>
+        <h2 id="welcome-sentence" class="welcome">
+          Can't wait when you add your first task!
+        </h2>
+      </ul>
+      <button id="sort-by-name-button" class="task-button hidden">
```

Comment:
you might want to wrap the sort & clear all buttons with a footer container to easier positioning